app/routes/user/password.tsx

```diff
@@ -0,0 +1,276 @@
+    }
+  }, [fetcher.state])
+
+  return (
+    <>
+      <h1>Reset Password</h1>
+      <>
+        <fetcher.Form method="POST" ref={formRef}>
+          <Label htmlFor="oldPassword">Old Password</Label>
+          <TextInput
+            data-focus
+            className={oldPasswordError ? 'usa-input--error' : ''}
+            name="oldPassword"
+            id="oldPassword"
+            type="password"
+            placeholder="Old Password"
+            onChange={(e) => {
+              setIsOldPasswordTouched(true)
+            }}
+          />
+          {errorMessage === 'NotAuthorizedException' && (
+            <span className="text-red">Invalid Password</span>
+          )}
+          {errorMessage === 'LimitExceededException' && (
+            <span className="text-red">
+              Attempts Exceeded. Please try again later.
+            </span>
+          )}
+          <Label htmlFor="newPassword">New Password</Label>
+          <InputGroup error={isError}>
+            <TextInput
+              name="newPassword"
+              id="newPassword"
+              className={
+                !isError && isNewPasswordTouched ? 'usa-input--success' : ''
+              }
+              type={visible ? 'text' : 'password'}
+              placeholder="New Password"
+              onChange={(e) => {
+                setNewPassword(e.target.value)
+                setIsNewPasswordTouched(true)
+              }}
+              value={newPassword}
+            />
+            {visible && (
+              <div className="usa-input-suffix" aria-hidden="true">
+                <Icon.VisibilityOff
+                  className="margin-y-2"
+                  onClick={() => {
+                    setVisible(!visible)
+                  }}
+                />
+              </div>
+            )}
+            {!visible && (
+              <div className="usa-input-suffix" aria-hidden="true">
+                <Icon.Visibility
+                  className="margin-y-2"
+                  onClick={() => {
+                    setVisible(!visible)
+                  }}
+                />
+              </div>
+            )}
+          </InputGroup>
+          <ButtonGroup>
+            <Button
+              disabled={shouldDisableSubmit}
+              type="submit"
+              className="margin-y-2"
+            >
+              Reset Password
+            </Button>
+          </ButtonGroup>
+          {fetcher.state !== 'idle' && (
+            <>
+              <Spinner /> Saving...
+            </>
+          )}
+          {fetcher.state === 'idle' &&
+            fetcher.data !== undefined &&
+            fetcher.data !== 'NotAuthorizedException' &&
+            fetcher.data !== 'LimitExceededException' && (
+              <>
+                <Icon.Check color="green" /> Saved
+              </>
+            )}
+        </fetcher.Form>
+      </>
+
+      <div className="usa-alert usa-alert--info usa-alert--validation">
```

Comment:
I believe that there is a react-uswds component for the USWDS alert box.

Reply:
The down side to the react-uswds component Alert is that it renders a p tag for the alert text and ul can't be a child of p. 

<img width="1202" alt="Screen Shot 2023-07-17 at 2 25 13 PM" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/6545874/0352f6e0-6b3b-4c50-b551-6fdf1b2319a4">

<img width="1404" alt="Screen Shot 2023-07-17 at 2 22 22 PM" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/6545874/ce45a1d3-4670-459d-81a8-5beaf4895deb">

under the hood it does the same thing that I had there, but instead of a p tag, I can use a div and there are no errors.

Reply:
No, your current solution is fine. Thanks for explaining!